test/conformance/chainsaw/exceptions/exclude-image-reference/policy.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-image
+spec:
+  validationFailureAction: Enforce
```

Comment:
This field is deprecated. You can use the `verifyImage[0].failureAction` instead.